phpBB/styles/prosilver/theme/common.css

```diff
@@ -1276,13 +1325,15 @@ ul.linklist:after,
 	padding: 8px;
 }
 
-.dropdown-extended .footer > a {
+.dropdown-extended .footer > a,
+.smiley-content .footer  > a {
 	padding: 5px 0;
 }
 
 .dropdown-extended ul li a,
 .notification_list dt > a,
-.dropdown-extended .footer > a {
+.dropdown-extended .footer > a,
+.smiley-content .footer  > a {
 	text-decoration: none;
```

Comment:
Same name cannot be added. Because  dropdown-extended class is used for the notification dropdown. 
Border colors of the pointers should be changed if the dropdown-extended class is used. Earliar I have used that. But now I removed that because in earliar pull request there was a request remove that border color styling.
In here we just need to add styles for the footer.